pytng/pytng.pyx

```diff
@@ -264,6 +331,64 @@ cdef class TNGFile:
         self.step += 1
         return TNGFrame(xyz, time, self.step - 1, box)
 
+    def _init_first_frame_write(self, int64_t n_atoms,
+                                box_shape_interval=1, pos_interval=1,
+                                time=None):
+        cdef int64_t ok
+
+        ok = tng_implicit_num_particles_set(self._traj, n_atoms)
+        if_not_ok(ok, 'Could not set the number of particles')
+
+        ok = tng_util_pos_write_interval_set(self._traj, 1)
+        if_not_ok(ok, 'Could not set position write interval')
+        ok = tng_util_box_shape_write_interval_set(self._traj, 1)
```

Comment:
Finally, this is needed. It seems that the interval is not 1 per default. As a result, not setting it (or setting it to anything else than 1) leads to only part of the frames being written.